sound/soc/sof/topology.c

```diff
@@ -1193,8 +1195,9 @@ static int sof_widget_load_pga(struct snd_soc_component *scomp, int index,
 	const struct snd_kcontrol_new *kc = NULL;
 	struct soc_mixer_control *sm;
 	struct snd_sof_control *scontrol;
+	struct sof_ipc_ctrl_data *cdata;
 	const unsigned int *p;
-	int ret, tlv[TLV_ITEMS];
+	int ret, i, tlv[TLV_ITEMS];
```

Comment:
nitpick: kernel reviewers like xmas-tree declarations and variables on different lines.

Reply:
ok will fix